synthea-hiv/uploader/fhir_client.py

```diff
@@ -45,18 +45,18 @@ class OpenMrsClient:
 
   def __init__(self, base_url: str):
     self.base_url = base_url
-    self.session = _setup_session(self.base_url)
```

Comment:
Why are you dropping `session` for this client but keeping it for the GCP version?